api/proto/info/info.proto

```diff
@@ -394,6 +395,38 @@ message ZCellularProvider {
   bool roaming = 4;
 }
 
+// Snapshot states
+enum ZSnapshotState {
+  Z_SNAPSHOT_STATE_UNSPECIFIED = 0;
+  // This state is used when a snapshot is in the process of being
+  // created or an error occurred during the first attempt to create it.
+  // (For example, the operation was delayed)
+  Z_SNAPSHOT_STATE_CREATING = 1;
+  // This state is used when the snapshot has been successfully created.
+  Z_SNAPSHOT_STATE_CREATED = 2;
+  // This state is used when the snapshot is pending deletion or
+  // the first deletion attempt was not successful.
```

Comment:
How about adding a comment saying that once the snapshot has been deleted it will not longer be reported as a ZInfoSnapshot?